test/dotnet/Unit/RedisPubSubTriggerBindingProviderTests.cs

```diff
@@ -2,17 +2,18 @@
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Logging;
 using System;
+using System.Threading.Tasks;
 using Xunit;
 
 namespace Microsoft.Azure.WebJobs.Extensions.Redis.Tests.Unit
 {
     public class RedisTriggerBindingProviderTests
     {
         [Fact]
-        public void Constructor_NullContext_ThrowsException()
+        public async Task Constructor_NullContext_ThrowsExceptionAsync()
```

Comment:
This doesn't really relate to updating dependencies. Should it be a separate PR?